src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -134,49 +91,71 @@ def test_WINNF_FT_S_FAD_1(self):
+                for record in data['recordData']:
+                # Verify the response files with CbsdData.schema.json Object schema
+                    self.assertContainsRequiredFields("CbsdData.schema.json", record)
+                    self.assertTrue(record['registration']['fccId'] in (device_a['fccId'], device_c['fccId']))
+                # Verify all the previous activities on CBSDs and Grants exist in the dump files    
+                for index, device in enumerate([device_a, device_c]):
+                    record_id = 'cbsd/'+ device['fccId']+'/'+ hashlib.sha1(device['cbsdSerialNumber']).hexdigest()
+                    cbsd_record = [record['registration'] for record in data['recordData'] if record['id'] == record_id]
+                    self.assertEqual(first_cbsd['cbsdCategory'],second_cbsd['cbsdCategory'])
+                    self.assertEqual(device['airInterface']['radioTechnology'],\
+                                     cbsd_record['airInterface']['radioTechnology'])
+                    self.assertEqual(device['installationParam']['latitude'],\
+                                     cbsd_record['installationParam']['latitude'])
+                    self.assertEqual(device['installationParam']['longitude'],\
+                                     cbsd_record['installationParam']['longitude'])
+                    self.assertEqual(device['installationParam']['height'],\
+                                     cbsd_record['installationParam']['height'])
+                    self.assertEqual(device['installationParam']['heightType'], \
+                                     cbsd_record['installationParam']['heightType'])
+                    self.assertEqual(device['installationParam']['indoorDeployment'], \
+                                     cbsd_record['installationParam']['indoorDeployment'])
+                    self.assertEqual(device['installationParam']['antennaGain'], \
+                                     cbsd_record['installationParam']['antennaGain'])                           
+                    # Get grants by cbsd_id
+                    grants_of_cbsd = [cbsd['grants'] for cbsd in data['recordData'] if cbsd['id'] == record_id]
+                    self.assertEqual(grants_of_cbsd[0][0]['id'], (grant_ids[index]))
+                    self.assertGrantRecord(record['grants'][0], grants_to_request[index], grant_response[index])
+                    # Verify the Grant Of the Cbsd
+                    self.assertEqual(record['grants'][0]['id'],grant_response[index]['grantId'])
+                    if record['grants'][0]['operationParam'] != null :
+                        self.assertEqual(record['grants'][0]['operationParam']['maxEirp'],\
+                                         grants_to_request[index]['operationParam']['maxEirp'])    
+                        self.assertEqual(record['grants'][0]['operationParam'], ['operationFrequencyRange']['lowFrequency'],\
```

Comment:
I think there's a cut-and-paste error here as assertEqual() only takes two arguments (you've given three) and the second argument seems to be missing its variable.